libs/shared/ui/src/lib/components/modals/modal-confirmation/modal-confirmation.tsx

```diff
@@ -101,6 +93,14 @@ export function ModalConfirmation({
           )}
         />
         {children}
+        {warning && (
+          <Callout.Root className="mb-2" color="yellow">
```

Comment:
```suggestion
          <Callout.Root className="mb-5" color="yellow">
```

I think the margin should be iso with the top, I don't find the design on Figma I think Yael do this

<img width="590" alt="Screenshot 2024-06-13 at 12 20 10" src="https://github.com/Qovery/console/assets/533928/2c293310-f6d8-4970-b819-85d3da4b35c5">